starfish/experiment/experiment.py

```diff
@@ -20,12 +20,16 @@ class FieldOfView:
 
     Auxiliary images can be accessed using a key, i.e., FOV[aux_image_type].
 
-    Properties
-    -------
-    name                   The name of the FOV.  In an experiment with only a single FOV, this may
-                           be None.
-    primary_image          The primary image for this field of view.
-    auxiliary_image_types  A set of all the auxiliary image types.
+    Attributes
+    ----------
+    name : str
+        The name of the FOV.  In an experiment with only a single FOV, this may be None.
```

Comment:
Second sentence should be removed.

Reply:
This is corrected in #701, so don't bother fixing it. :)

Reply:
To be clear: this comment doesn't need to be addressed?